lighthouse-core/audits/audit.js

```diff
@@ -85,6 +95,7 @@ class Audit {
       debugString: result.debugString,
       optimalValue: result.optimalValue,
       extendedInfo: result.extendedInfo,
+      renderer: audit.meta.category === 'Accessibility' ? Audit.RENDERERS.GROUP_BY_TAG : Audit.RENDERERS.GROUP_BY_STATUS,
```

Comment:
this line is a total hack to make a quick demo, would be more like the line below `audit.meta.renderer | Audit.RENDERERS.GROUP_BY_STATUS`